tests/test_modules.py

```diff
@@ -204,35 +205,36 @@ def test_install_profile_latest(self):
 
     def test_install_profile(self):
         self.test_enable_name_stream()
-        self.module_base.install(["httpd:2.4:1/default"])
+        self.module_base.install(["httpd:2.4:2/default"])
```

Comment:
May I ask you whu have you changed the argument `"httpd:2.4:1/default"` to `"httpd:2.4:2/default"`

Reply:
Leaving it as `httpd:2.4:1/default` does not work. This test calls `test_enable_name_stream`, which calls `self.module_base.enable(["httpd:2.4"])`, which adds a request to install oneof `'httpd:2.4:1::', 'httpd:2.4:2::'`. So the solver sees two install oneofs:

```
job install oneof httpd:2.4:NoRequires-1.noarch@_all httpd:2.4:NoRequires-2.noarch@_all [forcebest,setevr,setarch]
job install oneof httpd:2.4:NoRequires-1.noarch@_all [forcebest,setevr,setarch]
```

and produces the following problems:

```
problem 70e5c809 info package httpd:2.4:NoRequires-2.noarch conflicts with module(httpd) provided by httpd:2.4:NoRequires-1.noarch
problem 70e5c809 solution 154300cc deljob install oneof httpd:2.4:NoRequires-1.noarch@_all [forcebest,setevr,setarch]
problem 70e5c809 solution 2d67ee55 deljob install oneof httpd:2.4:NoRequires-1.noarch@_all httpd:2.4:NoRequires-2.noarch@_all [forcebest,setevr,setarch]
```

Reply:
Oh I see.
I think we can remove ` self.test_enable_name_stream()`. The line ` self.module_base.install(["httpd:2.4:2/default"])` also enable `module httpd:2.4`. The problem is that test does not state what starting condition is expected for the test.

I see following scenarios:
 - install a profile from a module stream that is not yet enabled
 - install a profile from a module stream that is already enabled (`using  self.base._moduleContainer.enable("httpd", "2.4", False)`)
 - install a profile from not the latest module item from a module stream that is not yet enabled